static/js/containers/pages/settings/AccountSettingsPage.js

```diff
@@ -123,7 +123,7 @@ export class AccountSettingsPage extends React.Component<Props> {
       >
         <div className="container auth-page account-settings-page">
           <div className="auth-header">
-            <h1>User Settings</h1>
```

Comment:
I don't think this change is necessary, and I have an open PR that updates this page layout anyway (#185). I think you can revert this change